.changeset/eleven-parents-brake.md

```diff
@@ -0,0 +1,8 @@
+---
+'@ai-sdk/provider-utils': patch
```

Comment:
rm provider utils since it's a test only change and provider utils changes trigger new releases for all providers